pyproject.toml

```diff
@@ -37,16 +37,19 @@ httpx = [
   { version = "^0.25.0", python = "^3.8" },
 ]
 h2 = "^4.1.0" # required for httx package, HTTP2 communication
-
-# Optional dependencies
-pycrypto = { version = "^2.6.1", optional = true }
-pycryptodome = { version = "*", optional = true }
-websockets = "^10.3"
+websockets = [
+  { version = "^11.0", python = "~3.7" },
```

Comment:
Can we preserve support for older versions of `websockets`?

Reply:
Actually, I gave a thought on this !
Seems, there will be lot of python packages that will use latest version of `websockets`. So, current `ably-python` won't be able to resolve it. `websockets 11.0` was released almost 9 months back ( pretty old ), so makes sense to upgrade it.

In a project where other packages are using `websockets ^10.0`, poetry will automatically install `ably-python 2.0.4` ( current version of the SDK ), so there won't be package conflicts as such.

Reply:
Also, sadly `python interpreter` doesn't allow multiple versions of `python package` in the same project.

Reply:
We are doing the same for `httpx` 
https://github.com/ably/ably-python/blob/aa5a963f6dea8da291b0d8aedcdc3790e742e7da/pyproject.toml#L35-L38

and pyee
https://github.com/ably/ably-python/blob/aa5a963f6dea8da291b0d8aedcdc3790e742e7da/pyproject.toml#L45-L49

Reply:
Sorry, I don't understand what is wrong with `version = ">=10.3,<12"`. Why we want to left part of users unsupported? I believe 9 months is pretty new package

Reply:
9 months really isn't that old in terms of library dependencies (for example most of the ably-python dependencies are on versions more than 9 months old), so I wouldn't be surprised if there were still many libraries out there using websockets ^10.0 and this change would make ably incompatible with them. Is it possible with poetry to just have a dependency spec spanning multiple versions, eg `>= 10.0, <= 12.0`?

Reply:
unfortunately, if we've done something before it doesn't automatically make it right

Reply:
I need to check if `>=10.3,<12` is feasible.

Reply:
Updated !